src/crispy/Note.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include <array>
+#include <cstdint>
+#include <vector>
+namespace crispy
+{
+enum class Note
```

Comment:
The class and the filename could probably be better named `MusicalNote`. I personally find `Note` a little bit too synonym-rich. :)

Reply:
Done